### ISSUE TITLE
bug(MatSelect): With VoiceOver + Chrome, "space" is announced when user navigates to mat-select if there is nothing yet selected from dropdown

### ISSUE THREAD

**Issue by rehoulihan** [angular/components#24731]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

If there is nothing selected from a mat-select when a user navigates to it with Voiceover tool, the Voiceover tool announces the selector as "space, {{name of mat-select}}". 

### Reproduction

Steps to reproduce:

1. Access https://material.angular.io/components/select/overview
2. Turn on Voiceover and navigate to Basic mat-select with tab
3. Observe Voiceover announces the selector as "**space**, Favourite food"

### Expected Behavior

Mat-select does not have 'space' announced before the name of the select. 

### Actual Behavior

Mat-select does have 'space' announced before the name of the select. 

### Environment

Angular: v11, v12, v13 (observed from the latest 3 versions on the official website)
CDK/Material: Material
Browser(s): Chrome
Operating System (e.g. Windows, macOS, Ubuntu): macOS
